src/renderer/components/workspaces/floating-panels/FloatingPanelLayer.tsx

```
@@ -186,73 +187,75 @@ function FloatingPanelWindow({
       >
-        <div onContextMenu={(event) => onContextMenu(panel.id, entry.id, event)}>
-          <GenericPanel
-          title={title}
-          onClose={() => removeFloatingPanel(scope, panel.id)}
-          storageKey={`floating-panel:${scope}:${panel.id}`}
-          initialPosition={panel.initialPosition}
-          initialSize={panel.initialSize}
-          minSize={panel.minSize}
+        <div
+          onContextMenu={(event) => onContextMenu(panel.id, entry.id, event)}
         >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)